src/routes/Home.tsx

```diff
@@ -217,7 +213,9 @@ const Home = () => {
               <h1 className="text-3xl font-bold">Your Projects</h1>
               <ActionButton
                 Element="button"
-                onClick={() => send('Create project')}
+                onClick={
+                  () => send({ type: 'Create project', data: { name: '' } }) // TODO: @frank to check this
```

Comment:
@franknoirot Can you check if passing empty string here is safe?

Reply:
It wasn't but only because we didn't check if `name` was falsy, and I've added that check in now.